spec/models/photo_uploader_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'carrierwave/test/matchers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.